website/docs/d/repositories.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "github"
+page_title: "GitHub: github_repositories"
+sidebar_current: "docs-github-datasource-repositories"
+description: |-
+  Search for GitHub repositories
+---
+
+# github_repositories
+
+-> **Note:** The data source will return a maximum of `1000` repositories
+	[as documented in official API docs](https://developer.github.com/v3/search/#about-the-search-api).
+
+Use this data source to retrieve a list of GitHub repositories using a search query.
+
+## Example Usage
+
+```hcl
+data "github_repositories" "example" {
+  query = "org:hashicorp language:Go"
```

Comment:
I'd question whether we want to also expose these as properties on the data source (e.g. `org` / `language`)?

Reply:
nope, I was just thinking out loud :)